hudi-common/src/main/java/org/apache/hudi/common/table/TableSchemaResolver.java

```diff
@@ -526,6 +526,7 @@ private boolean hasOperationField() {
       Schema tableAvroSchema = getTableAvroSchemaFromDataFile();
       return tableAvroSchema.getField(HoodieRecord.OPERATION_METADATA_FIELD) != null;
     } catch (Exception e) {
+      LOG.info(String.format("Failed to read operation field from avro schema (%s)", e.getMessage()));
```

Comment:
i recommend there is no logging. It'll affect the user experience. See the logging of CI, you can find this.

Reply:
I've removed the exception logging, so it should not be noisy. I still do think that we should keep at least 1-line log-statement that something failed hence we fallback

Reply:
can we make this error log level instead of info?

Reply:
I would rather reserve "error" level for things that flag malfunction and as such require immediate attention. This is just referring that we're trying to fetch particular failed that wasn't found